### PR TITLE
CedricLouvrier.Pingo version 1.23.9

### PR DESCRIPTION
**Pull request by ye4241**
Checklist for Pull Requests
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Is there a linked Issue?

Manifests
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/doc/Authoring.md#validation) your manifest locally with `winget validate --manifest .\manifests\c\CedricLouvrier\Pingo\1.23.9\`?
- [x] Have you tested your manifest locally with `winget install --manifest .\manifests\c\CedricLouvrier\Pingo\1.23.9\`?
- [x] Does your manifest conform to the [1.6 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.6.0)?

Note: `<path>` is the directory's name containing the manifest you're submitting.

---

### CODE REVIEW

**Comment by ye4241** [comment]
The download URL requires User-Agent verification, which prevents direct downloads from succeeding. Therefore, I used CloudFlare Workers to create a reverse proxy for the website's download link, allowing the installer file to be downloaded correctly.

**Comment by stephengillie** [comment]
> The download URL requires User-Agent verification, which prevents direct downloads from succeeding. Therefore, I used CloudFlare Workers to create a reverse proxy for the website's download link, allowing the installer file to be downloaded correctly.

Hi @ye4241,

That's quite the configuration of services. Unfortunately, packages really need to be provided by their developer, and we can't rely on 3rd party hosting - or in this case, intermediation. Especially if the developer is blocking based on user-agent string. A package manager should facilitate the relationship between developer and user, not circumvent it. 

Unfortunately, this package might not be a good fit for this package manager, at least until the blocking situation changes.

File: manifests/c/CedricLouvrier/Pingo/1.23.9/CedricLouvrier.Pingo.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+PackageIdentifier: CedricLouvrier.Pingo
+PackageVersion: 1.23.9
+InstallerType: zip
+NestedInstallerType: portable
+NestedInstallerFiles:
+- RelativeFilePath: pingo.exe
+Installers:
+- Architecture: x64
+  InstallerUrl: https://css-ig.ye4241.workers.dev/bin/pingo-win64.zip
```
Comment (ItzLevvie) [inline manifests/c/CedricLouvrier/Pingo/1.23.9/CedricLouvrier.Pingo.installer.yaml:12]:
```suggestion
  InstallerUrl: https://css-ig.net/bin/pingo-win64.zip
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Comment by ItzLevvie** [comment]
[Policy] Needs-Author-Feedback

**Comment by ItzLevvie** [comment]
[Policy] Blocking-Issue

File: manifests/c/CedricLouvrier/Pingo/1.23.9/CedricLouvrier.Pingo.installer.yaml (hunk 1 above)
Comment (ye4241) [reply manifests/c/CedricLouvrier/Pingo/1.23.9/CedricLouvrier.Pingo.installer.yaml:12]: CleanShot 2024-09-07 at 21 40 33@2x

This link needs JavaScript to download real zip file, therefore, I use CloudFlare to resolve this issue.

**Comment by denelon** [comment]
This PR is using an older version of the schema. I'm going to go ahead and close this for now. Once the manifest has been updated, either create a new PR or re-open this one.

**Comment by ye4241** [comment]
Reopen it, I have reverse installer url to official link.

**Comment by ye4241** [comment]
@denolon Please reopen it, I have reverse installer url to official link.

**Pull request closed without merging**